import-export-cli/integration/testutils/operationPolicy_testUtils.go

```diff
@@ -0,0 +1,552 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package testutils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/apim"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/base"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	PolicyVersionKey = "version"
+	PolicyNameKey    = "name"
+	PolicyIDReset    = "-1"
+	PolicyIdKey      = "id"
+)
+
+const (
+	DefaultPolicyListSize       = "-1"
+	DefaultPolicyListOffsetSize = "-1"
+	DefaultPolicyListLimit      = "25"
+	DefaultPolicyListOffset     = "0"
+)
+
+// ValidateAPIPolicyExportImport : Validates Exporting API Policy from source env and Importing to destination env
+func ValidateAPIPolicyExportImport(t *testing.T, args *PolicyImportExportTestArgs) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[PolicyNameKey])
+	policyVersion := fmt.Sprintf("%v", args.Policy[PolicyVersionKey])
+
+	args.SrcAPIM.Login(args.CtlUser.Username, args.CtlUser.Password)
+
+	exportedOutput, _ := exportAPIPolicy(t, policyName, args.SrcAPIM.GetEnvName())
+
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+
+	// fmt.Println("Exported Path: ", args.ImportFilePath)
```

Comment:
shall we remove this?

Reply:
Fixed by [159a8a5](https://github.com/wso2/product-apim-tooling/pull/925/commits/159a8a534b0a32c65e3b49e708a9079a179de22a)